src/xcode/ENA/ENA/Source/Scenes/Home/HomeInteractor.swift

```diff
@@ -219,7 +219,7 @@ extension HomeInteractor {
 
```

Comment:
Can riskDetails now be completely removed from the state, because I don't see any usage.

Reply:
Eagle eyes! 🦅 Have removed it, it was not used anymore.